hack/gen-api-reference-docs.sh

```diff
@@ -163,6 +164,8 @@ main() {
     log "Copy the files in reference/ directory to knative/docs."
     if command -v open >/dev/null; then
         open "${clone_root}/out/"
+    elif command -v xdg-open >/dev/null; then
```

Comment:
This failed for me on Linux. I searched and found this on stackoverflow: 
[On linux] "The open command is a link to the openvt command and opens a binary in a new virtual console." which is not what we want and why the conditional does not fail.

I tried swapping the commands so that 'xdg-open' is tried/run first, before 'open', and it worked. But since I dont have my Mac setup right now to test the script, i've only just tired running 'xdg-open', which results in "-bash: xdg-open: command not found" (so I think we should be okay?).